Details of RBAC for Input Sets [CDS-80218] (#5392)
harness/developer-hub: docs/platform/pipelines/input-sets.md

<!-- However, it is possible that you won't use every input set in the overlay for every run. When you run a pipeline with an overlay, you can select specific input sets to use. If an input set is not selected for a particular run, it is skipped. -->

> ## Manage Access to Input Sets

Reviewer: ```suggestion
## Manage access to input sets
```

> :::tip

Reviewer: ```suggestion
:::info
```

> Input Sets can be access controlled, similar to Pipelines and other entities. This allows you to make some input sets invisible to certain users, or not let them edit the values in the Input Sets. Input Sets are listed along with other resources in the "Resource Group" section of the Access Control screens. For more information on setting up and managing permissions using Resource Groups, see [Managing Resource Groups](add-resource-groups).

Reviewer: ```suggestion
Input sets can be access-controlled, similar to pipelines and other entities. This allows you to make some input sets invisible to certain users, or not let them edit the values in the input sets. Input sets are listed along with other resources in the **Resource Group** section of the Access Control screens. For more information on setting up and managing permissions using Resource Groups, go to [Manage resource groups](/docs/platform/role-based-access-control/add-resource-groups/).
```

> In order to use an Input Set for a Pipeline Execute, the user must have View permissions on the Input Set, along with Execute permissions on the Pipeline itself. In order to edit the Input Set, she must have Edit permissions on it. 

Reviewer: ```suggestion
To use an input set for a pipeline execution, the user must have View permissions on the input set, along with Execute permissions for the pipeline. To edit the input set,  the user must have Edit permissions for the pipeline. 
```

> When a user creates a new Input Set, she by default has the Create/Edit and View permissions on it. 

Reviewer: ```suggestion
When a user creates a new input set, the Create/Edit and View permissions are added by default for the user. 
```

:::tip

> Access Control to Input Sets is enabled using the Feature Flag PIE_INPUTSET_RBAC_PERMISSIONS. Please contact Harness Support to enable it, if not available in your account.

Reviewer: ```suggestion
Currently, Input set access control is behind the feature flag `PIE_INPUTSET_RBAC_PERMISSIONS`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
```

<!-- However, it is possible that you won't use every input set in the overlay for every run. When you run a pipeline with an overlay, you can select specific input sets to use. If an input set is not selected for a particular run, it is skipped. -->

## Manage access to input sets

:::info

> Currently, Input set access control is behind the feature flag `PIE_INPUTSET_RBAC_PERMISSIONS`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. We will also run a migration for you so that existing Input Sets remain accessible to users.

Reviewer: ```suggestion
Currently, input set access control is behind the feature flag `PIE_INPUTSET_RBAC_PERMISSIONS`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. We will also run a migration for you so that existing input sets remain accessible to users.
```

:::

> Input sets can be access-controlled, similar to pipelines and other entities. This allows you to make some input sets invisible to certain users, or not let them edit the values in the input sets. The permissions for Input Sets are View, Create/Edit and Delete. These permissions can be viewed and managed in the **Roles** page under Access Control.

Reviewer: ```suggestion
Input sets can be access-controlled, similar to pipelines and other entities. This allows you to make some input sets invisible to certain users, or not let them edit the values in the input sets. The permissions for input sets are View, Create/Edit, and Delete. You can view and manage these permissions on the Roles page under **Access Control**.
```

Input sets are listed along with other resources in the **Resource Group** section of the Access Control screens. For more information on setting up and managing permissions using Resource Groups and Roles, go to [Manage resource groups](/docs/platform/role-based-access-control/add-resource-groups/).

> To use an input set for a pipeline execution, the user must have View permissions on the input set, along with Execute permissions for the pipeline. To edit the input set, the user must have Edit permissions for the Input Set. 

Reviewer: ```suggestion
To use an input set for a pipeline execution, the user must have View permissions on the input set, along with Execute permissions for the pipeline. To edit the input set, the user must have Edit permissions for the input set. 
```

> When a user creates a new input set, the Create/Edit and View permissions are added by default for the user. 

Reviewer: ```suggestion
When a user creates a new input set, Create/Edit and View permissions are added by default for the user. 
```
